### PR TITLE
updated tcpdump, trufflehog, viper and whois for HUB

### PR DESCRIPTION
**Pull request by tylerslijboom-r7**
### Description

UPdated help.md and spec for Hub as per google doc

### Testing

ICONVALIDATED

### CODE REVIEW

File: trufflehog/bin/komand_trufflehog [hunk 1]
```diff
@@ -4,10 +4,10 @@ import komand
...
 
 
-Name = 'TruffleHog'
-Vendor = 'rapid7'
-Version = '1.1.2'
-Description = 'Searche through git repositories for high entropy strings and secrets, digging deep into commit history'
+Name = "TruffleHog"
+Vendor = "rapid7"
+Version = "1.1.3"
+Description = "Searche through git repositories for high entropy strings and secrets, digging deep into commit history"
```
Comment (lmilby-r7) [inline trufflehog/bin/komand_trufflehog]:
```suggestion
Description = "Searches through git repositories for high entropy strings and secrets, digging deep into commit history"
```

File: trufflehog/help.md [hunk 2]
```diff
@@ -1,18 +1,15 @@
 # Description
 
-The [TruffleHog](https://github.com/dxa4481/truffleHog) plugin uses regex and entropy checks to go through the entire commit history of each branch, and check each diff for every commit as well as for secrets.
-
-For entropy checks, TruffleHog will evaluate the shannon entropy for both the base64 character set and hexadecimal character set for every blob of text greater than 20 characters comprised of those character sets in each diff.
-If at any point a high entropy string is greater than 20 characters is detected, it will collect the hashes and provide them for use once complete.
+Searche through git repositories for high entropy strings and secrets, digging deep into commit history
```
Comment (lmilby-r7) [inline trufflehog/help.md]:
```suggestion
Searches through git repositories for high entropy strings and secrets, digging deep into commit history
```
Was there an issue with the existing description?

File: trufflehog/help.md [hunk 3]
```diff
@@ -1,18 +1,15 @@
...
 
 # Key Features
 
-* Search through github commit histories 
-* Check for leaked secrets
+Identify key features of plugin.
```
Comment (lmilby-r7) [inline trufflehog/help.md]:
```suggestion
* Identify key features of plugin
```

File: trufflehog/help.md [hunk 4]
```diff
@@ -1,18 +1,15 @@
...
 
 # Requirements
 
-_This plugin does not contain any requirements._
+* Example: Requires an API Key from the product
```
Comment (lmilby-r7) [inline trufflehog/help.md]: Revert to old requirements

File: trufflehog/help.md [hunk 5]
```diff
@@ -26,18 +23,18 @@ _This plugin does not contain a connection._
...
 |Name|Type|Default|Required|Description|Enum|
 |----|----|-------|--------|-----------|----|
-|git_url|string|All|True|The git repository that is going to be searched|None|
-|do_regex|boolean|False|False|Enable high signal regex checks|None|
+|custom_regexes|object|None|False|Ignores default regexes. Provide your own|None|
 |do_entropy|boolean|True|False|Evaluates the shannon entropy for both the base64 char set and hexadecimal char set for every blob of text greater than 20 characters comprised of those character sets in each diff|None|
-|custom_regexes|object|None|True|Ignores default regexes. Provide your own|None|
-|since_commit|string|None|False|Scan from a given commit hash|None|
+|do_regex|boolean|False|False|Enable high signal regex checks|None|
+|git_url|string|None|True|The git repository that is going to be searched e.g. https://github.com/jonschipp/islet|None|
```
Comment (lmilby-r7) [inline trufflehog/help.md]: fix the example url

File: trufflehog/help.md [hunk 6]
```diff
@@ -48,45 +45,6 @@ This action is used to search through github commit histories, and checks if the
 Example output:
 
 ```
-
```
Comment (lmilby-r7) [inline trufflehog/help.md]: missing example output

File: trufflehog/help.md [hunk 7]
```diff
@@ -112,5 +84,4 @@ _This plugin does not contain any troubleshooting information._
 
 ## References
 
-* [truffleHog](https://github.com/dxa4481/truffleHog)
-
+* [TruffleHog](LINK TO PRODUCT/VENDOR WEBSITE)
```
Comment (lmilby-r7) [inline trufflehog/help.md]:
```suggestion
* [TruffleHog](https://github.com/dxa4481/truffleHog)
```

File: trufflehog/help.md (hunk 3 above)
Comment (tylerslijboom-r7) [reply trufflehog/help.md]: Actually, this also needs to be an actual feature of the plugin!

File: trufflehog/help.md (hunk 6 above)
Comment (tylerslijboom-r7) [reply trufflehog/help.md]: added

File: tcpdump/plugin.spec.yaml [hunk 8]
```diff
@@ -38,7 +38,7 @@ actions:
         required: false
       filter:
         title: BPF
-        description: Berkely Packet Filter E.g. tcp and port 22
+        description: Berkely Packet Filter E.g. TCP and port 22
```
Comment (mrinehart-r7) [inline tcpdump/plugin.spec.yaml]:
```suggestion
        description: Berkeley Packet Filter E.g. TCP and port 22
```

**Pull request merged**